types/office-js-preview/index.d.ts

```diff
@@ -15569,6 +15600,25 @@ declare namespace Office {
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Message Compose
          */
         conversationId: string;
+        /**
+         * Gets the Base64-encoded position of the current message in a conversation thread.
+         *
+         * @remarks
+         * [Api set: Mailbox preview]
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Message Compose
+         *
+         * **Important**: To preview the `conversationIndex` property, you must install Outlook on Windows Version 2402 (Build 17317.20000) or later. Then, join the
+         * {@link https://insider.microsoft365.com/join/Windows | Microsoft 365 Insider program} and select the **Beta Channel** option to access Office beta builds.
+         *
+         * **Tip**: You can use the `conversationIndex` property value to locate a message in a conversation thread and use its contents
+         * to provide context for the current message being composed.
```

Comment:
Consider making this two sentences for easier parsing.
```suggestion
         * **Tip**: You can use the `conversationIndex` property value to locate a message in a conversation thread. 
         * Use its contents to provide context for the current message being composed.
```